src/Cosmos/ProtobufSerialization.cpp

```diff
@@ -302,7 +302,7 @@ Data buildSignature(const Proto::SigningInput& input, const std::string& seriali
     return signature;
 }
 
-std::string buildProtoTxRaw(const Proto::SigningInput& input, const std::string& serializedTxBody, const std::string& serializedAuthInfo, const Data& signature) {
+std::string buildProtoTxRaw([[maybe_unused]] const Proto::SigningInput& input, const std::string& serializedTxBody, const std::string& serializedAuthInfo, const Data& signature) {
```

Comment:
looks like we can remove it

Reply:
Agree, let's remove `input` parameter in this PR.

Reply:
will do

Reply:
fix in [8c26479](https://github.com/trustwallet/wallet-core/pull/2397/commits/8c26479da625ce0af181b6483e9ab7f97ebc2f79)